algebird-core/src/main/scala/com/twitter/algebird/DecayedValue.scala

```diff
@@ -73,4 +73,19 @@ case class DecayedValue(value: Double, scaledTime: Double) extends Ordered[Decay
     val normalization = halfLife / math.log(2)
     value / normalization
   }
+
+  /*
+  * Moving average assuming the signal started at zero a fixed point in the past.
+  * This normalizes by the integral of exp(-t(ln(2))/halflife) from 0 to (endTime - startTime).
+  */
+  def averageFrom(halfLife: Double, startTime: Double, endTime: Double) = {
```

Comment:
`averageFrom(h, 0.0, e)` can be > `average(h)` if `e < n*h` for small `n`. That's intended; the `average` goes to time `-Inifnity`, whereas `averageFrom` does not.

However, `averageFrom(h, 0.0, e)` will converge to `average(h)` as `e` goes to infinity, which I think is the actual property you want to test.